schema.ts

```diff
@@ -1245,6 +1246,18 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
    * @hidden
    */
   groupIdProperty?: PropertyIdentifier<K>;
+
+  /**
+   * The name of the property within {@link ObjectSchemaDefinition.properties} that can be be interpreted as
+   * a large blob of unstructured content.
+   *
+   * This should consist of text or markdown content.
```

Comment:
not HTML?

Reply:
In theory could be HTML. Depends how well our models will support this format

Reply:
String properties already support optional codaType hints of Markdown or Html. So I think we should just look to those hint types to determine how to interpret the text (and if no hint type is provided it is plain text).